index.html

```diff
@@ -26,7 +40,7 @@
         <div class="cell"></div>
         <div class="cell"></div>
       </div>
-      <div class="tile" data-row="r1", data-col="c1" data-val="2">2</div>
     </div>
+    <footer> Brandi & Brittany Production </footer>
```

Comment:
GitHub's syntax highlighter has marked this `&` in red to indicate that it expected an HTML entity. Here, you want `&amp;` for an ampersand.